src/queries.ts

```diff
@@ -84,6 +100,12 @@ async function sendQuery(query: Query): Promise<GQLTransactionsResultInterface>
 	return JSONBody.data.transactions as GQLTransactionsResultInterface;
 }
 
+/**
+ * Returns a query object to math all ArDrive transactions within a range of blocks
```

Comment:
Phrasing seems off here, I'm not sure what `to math all ArDrive transactions` means

Reply:
"match", not "math" :sweat_smile:

Reply:
448bb77f4881c1f7e1d837927cddab4f5edf4ce9